RosettaAnnotations/pom.xml

```diff
@@ -10,4 +10,10 @@
 
   <artifactId>RosettaAnnotations</artifactId>
 
+  <dependencies>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
```

Comment:
This makes `RosettaAnnotations` much less light-weight. I think the annotations that require this dependency should go in `RosettaCore` instead (similar the way `@JsonDeserialize` lives in jackson-databind rather than jackson-annotations)